src/components/DashboardHeader.tsx

```diff
@@ -31,7 +31,9 @@ const DashboardHeader = ({ onExit, loggedUser, parties }: Props) => {
   const products = useAppSelector(partiesSelectors.selectPartySelectedProducts);
   const selectedParty = useAppSelector(partiesSelectors.selectPartySelected);
 
-  const [openEnvironmentModal, setOpenEnvironmentModal] = useState<boolean>(false);
+  const [openCustomEnvInteropModal, setopenCustomEnvInteropModal] = useState<boolean>(false);
```

Comment:
setter function name should be in camelcase

Reply:
done